pdf_build_src/header.tex

```diff
@@ -2,5 +2,5 @@
 \usepackage{fancyhdr}
 \pagestyle{fancy}
 \fancyhf{}
-\chead{Brain Imaging Data Structure v1.2.1 2019-08-14}
+\chead{ Brain Imaging Data Structure v1.4.1-dev 2020-08-04 }
```

Comment:
are these changes (in header and cover .tex) needed? I thought these get automatically updated upon rendering and the present values are simply "placeholders"?

Reply:
Good catch. Yes, it seems they are indeed automatically rewritten by the add_header() and edit_titlepage() functions in the process_markdown.py script so I guess these changes are not needed.

Reply:
yes, please revert them :+1: as few unnecessary changes as possible :-)